### PR TITLE
Notes on Logix Conditional migration

### PR DESCRIPTION
**Pull request by silverailscolo**
also added Wouters name

### CODE REVIEW

File: releasenotes/jmri4.11.4.shtml [hunk 1]
```diff
@@ -189,7 +189,7 @@ These may be relevant to you if you're updating from an earlier version.
     cannot be automatically flagged because JMRI allows file: URLs; these will have to 
     be changed manually.
         
-<p><span class="since">Since <a href="jmri4.7.1.shtml">JMRI 4.7.1</a></span><a id="validate" name="validate"/>
+<p><span class="since">Since <a href="jmri4.7.1.shtml">JMRI 4.7.1</a></span><a id="validate" name="validate">
```
Comment (mattharris) [inline releasenotes/jmri4.11.4.shtml]: Looks to be missing a closing `</a>` at the end of this line.

File: releasenotes/jmri4.11.4.shtml [hunk 2]
```diff
@@ -543,13 +550,13 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=is%3Aclosed&q=milest
...
 
-   <h3>Dispatcher</h3>
+    <h3>Dispatcher</h3>
         <ul>
              <li></li>
         </ul>
 
     <h3>Layout Editor</h3>
-        <a id="LE" name="LE"/>
+        <a id="LE" name="LE">
```
Comment (mattharris) [inline releasenotes/jmri4.11.4.shtml]: Similar issue as above - missing closing `</a>`.

File: releasenotes/jmri4.11.4.shtml [hunk 3]
```diff
@@ -578,8 +585,12 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=is%3Aclosed&q=milest
         </ul>
 
     <h3>Logix</h3>
+    <a id="logix" name="logix">
```
Comment (mattharris) [inline releasenotes/jmri4.11.4.shtml]: Similar issue as above - missing closing `</a>`.

Comment (silverailscolo) [reply releasenotes/jmri4.11.4.shtml]: Thanks, fixed

**Pull request merged**